Form/FormFactory.php

```diff
@@ -95,24 +96,9 @@ public function createForm($name, $data = [], $options = [])
      * @return string
      * @throws InexistentFormException
      */
-    public function getJsonConfiguration()
+    public function getJsonConfiguration($name)
     {
-        $configuration = [];
-
-        foreach ($this->configuration as $formName => $fieldConfiguration) {
-            $fields = [];
-
-            foreach ($fieldConfiguration as $field => $options) {
-                if (!$options['enabled']) {
-                    continue;
-                }
-
-                $fields[$field] = $options;
-            }
-
-            $configuration[$formName] = $fields;
-        }
-
-        return json_encode($configuration);
+        $response = new JsonResponse($this->configuration[$name]);
+        return $response;
```

Comment:
The FormFactory is not responsible for handling HTTP stuff. You should return the JSON only.